.github/dependabot.yaml

```diff
@@ -0,0 +1,62 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    labels:
+      - dependencies
+      - team/agent-platform
+      - changelog/no-changelog
+      - qa/skip-qa
```

Comment:
I don't think dep updates should automatically skip QA, depends on which deps are updated.

Reply:
Fixed!